app/utils/form-data.js

```diff
@@ -0,0 +1,3 @@
+const userType = ['admin', 'responder', 'requestor'];
+
+export { userType };
```

Comment:
It should be more readable, and easier to know that it's a collection:
```suggestion
const userTypes = ['admin', 'responder', 'requestor'];

export { userTypes };
```